HPXMLtoOpenStudio/measure.rb

```diff
@@ -1296,11 +1305,13 @@ def self.add_foundation_walls_slabs(runner, model, spaces)
       slabs = []
       @hpxml.foundation_walls.each do |foundation_wall|
```

Comment:
By adding conditions to skip small surfaces, I noticed it's looping through hpxml elements multiple times here, thus makes me wonder it might be a better way to handle it. Maybe use a hash to map foundation_type with its assigned slab and foundation walls? And this can also be used in the error checking above?

Reply:
Good thinking. I will replace these two loops when I implement the more general error-checks to ensure spaces have minimum adjacent surface types, so it's fine for now.